sebs/benchmark.py

```diff
@@ -288,6 +301,16 @@ def add_deployment_files(self, output_dir):
         for file in handlers:
             shutil.copy2(file, os.path.join(output_dir))
 
+    def add_deployment_package_java(self, output_dir):
+        # append to the end of requirements file
+        packages = self._system_config.deployment_packages(
+            self._deployment_name, self.language_name
+        )
+        if len(packages):
+            with open(os.path.join(output_dir, "requirements.txt"), "a") as out:
```

Comment:
@mahlashrifi This does not look correct - `requirements.txt` should only be used for Python projects. In Python, we add packages needed for specific functions to this file such that `pip` installs them.

Shouldn't we use something else for Java?